### PR TITLE
ign -> gz Macro Migration : gz-sim

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/737

### CODE REVIEW

File: test/integration/log_system.cc [hunk 1]
```diff
@@ -1523,7 +1523,7 @@ TEST_F(LogSystemTest, IGN_UTILS_TEST_DISABLED_ON_WIN32(LogResources))
   // Recorded models should exist
   EXPECT_GT(entryCount(recordPath), 2);
   EXPECT_TRUE(common::exists(common::joinPaths(recordPath, homeFake,
-      ".ignition", "fuel", "fuel.ignitionrobotics.org", "openrobotics",
+      ".gz", "fuel", "fuel.ignitionrobotics.org", "openrobotics",
```
Comment (chapulina) [inline test/integration/log_system.cc:1526]: We shouldn't change these yet

**Review by chapulina — CHANGES_REQUESTED** [review]
LGTM besides the Fuel directories.

File: test/integration/log_system.cc (hunk 1 above)
Comment (methylDragon) [reply test/integration/log_system.cc]: [`a3ecb2b` (#1520)](https://github.com/gazebosim/gz-sim/pull/1520/commits/a3ecb2b9b170b743a89c459faba90884b22eb70d)

File: src/ign.hh [hunk 2]
```diff
@@ -21,7 +21,8 @@
 
 /// \brief External hook to read the library version.
 /// \return C-string representing the version. Ex.: 0.1.2
-extern "C" char *ignitionGazeboVersion();
+extern "C" char *gzSimVersion();
+extern "C" char GZ_DEPRECATED(7) *ignitionGazeboVersion();
```
Comment (chapulina) [inline src/ign.hh:25]: No need to tick-tock, it's not installed

**Review by chapulina — CHANGES_REQUESTED** [review]

File: src/ign.hh (hunk 2 above)
Comment (methylDragon) [reply src/ign.hh]: https://github.com/gazebosim/gz-sim/pull/1520/commits/b969c8bd754a44e9fe0aba50ae54a131ef015966

**Review by chapulina — APPROVED** [review]

**Pull request merged**